src/lib/components/footer/QFooter.svelte

```diff
@@ -2,26 +2,30 @@
   import { getContext } from "svelte";
   import type { QFooterProps } from "./props";
   import { createClasses, createStyles } from "$lib/utils/props";
-  import type { AppbarContext, LayoutContext } from "../layout/QLayout.svelte";
+  import type { LayoutContext } from "../layout/QLayout.svelte";
   import { useSize } from "$lib/composables/use-size";
 
   export let value: QFooterProps["value"] = true,
-    bordered: QFooterProps["bordered"] = false,
-    elevated: QFooterProps["elevated"] = false,
+    border: QFooterProps["border"] = false,
```

Comment:
It appears these should be `bordered` and `elevated`, not sure why they were changed.